corehq/messaging/smsbackends/telerivet/tests/test_log_call.py

```diff
@@ -1,10 +1,10 @@
-from corehq.apps.ivr.tests.util import LogCallTestCase
+import corehq.apps.ivr.tests.util as util
```

Comment:
I don't understand it really, but for some reason the old import was causing `LogCallTestCase` to appear as if it was defined inside this module, which was causing [this conditional](https://github.com/dimagi/commcare-hq/blob/a7963baf03e7de2f2124de7c7d229487a9338202/corehq/apps/ivr/tests/util.py#L55) to fail.

see also: commit message on https://github.com/dimagi/commcare-hq/commit/a7963baf03e7de2f2124de7c7d229487a9338202